schema.dart

```diff
@@ -489,3 +495,80 @@ enum IOSElementType {
   touchBar,
   statusItem,
 }
+
+enum GoogleApp {
+  calculator('com.google.android.calculator'),
+  calendar('com.google.android.calendar'),
+  chrome('com.android.chrome'),
+  drive('com.google.android.apps.docs'),
+  gmail('com.google.android.gm'),
+  maps('com.google.android.apps.maps'),
+  photos('com.google.android.apps.photos'),
+  playStore('com.android.vending'),
+  settings('com.android.settings'),
+  youtube('com.google.android.youtube');
```

Comment:
Let’s add a comment about the source from which we took these strings.

Reply:
Android ones were fetched manually since I found no official doc from Google with those package names. Do you think linking the tool makes sense?
https://play.google.com/store/apps/details?id=com.csdroid.pkg&pcampaignid=web_share

Reply:
Definitely